MaxText/train.py

```diff
@@ -935,6 +935,9 @@ def train_loop(config, state=None):
         jax.block_until_ready(state)  # Block until current state finishes to end profile cleanly
       prof.deactivate()
 
+    if step == start_step:
+      max_utils.print_mem_stats(label=config.run_name)
```

Comment:
Can we change label here to "After params initalized" as well like https://github.com/AI-Hypercomputer/maxtext/blob/6ec3368af31fff6e6d735ac9d5fb77f91fc0f784/MaxText/maxengine.py#L156?

For reference this is only showing the HBM usage of the optimizer state. This can help us verify that host offloading is working as expected

Reply:
Thanks for the suggestion and done!